Sources/PodToBUILD/GlobUtils.swift

```diff
@@ -127,20 +126,25 @@ public class Glob: Collection {
         let fileManager = FileManager.default
 
         var directories: [String]
-
-        do {
-            directories = try fileManager.subpathsOfDirectory(atPath: firstPart).compactMap { subpath in
-                let fullPath = NSString(string: firstPart).appendingPathComponent(subpath)
-                var isDirectory = ObjCBool(false)
-                if fileManager.fileExists(atPath: fullPath, isDirectory: &isDirectory) && isDirectory.boolValue {
-                    return fullPath
-                } else {
-                    return nil
+        // If using an empty filepath then no need to look for it
+        if !firstPart.isEmpty {
+            do {
+                directories = try fileManager.subpathsOfDirectory(atPath: firstPart).compactMap { subpath in
```

Comment:
I've also hit a few cases where this starts to fail, I'm not sure what the behavior was before - e.g. was using "" equal to using "."?